Rakefile

```diff
@@ -184,28 +184,31 @@ task "coverage" => [:coverage_spec]
   task "frozen_#{task_suffix}" do
     block.call("CLOVER_FREEZE" => "1")
   end
+end
 
-  desc "Run specs#{desc_suffix} with coverage"
-  task "coverage_#{task_suffix}" do
-    block.call("COVERAGE" => "1")
-    if task_suffix == "pspec" || (task_suffix == "spec" && auto_parallel_tests.call)
-      # turbo_tests with coverage does not fail if coverage is not 100%,
-      # because it forks and each worker until the last is likely to have
-      # failing tests.  Check the generated coverage output after specs run to
-      # see if it is 100%.
-      coverage_output = File.binread("coverage/index.html")
-
-      # Results for the entire run are before the results for individual files,
-      # so this will get the results for the entire run.
-      lines_missed = %r{<b>(\d+)</b> lines missed\.}.match(coverage_output)[1]
-      branches_missed = %r{<b>(\d+)</b> branches missed\.}.match(coverage_output)[1]
-
-      unless lines_missed == "0" && branches_missed == "0"
-        warn "SimpleCov failed with exit 2 due to a coverage related error"
-        exit(2)
-      end
-    end
+desc "Run specs in serial with coverage"
```

Comment:
Is it intentional to have two tasks with an identical description?

Reply:
No, that's not intentional.  I'll fix it.  Thank you for pointing it out.

Reply:
I fixed this an rebased on main.